File: pset6/mario/less/mario.py
from cs50 import get_int

def main():
    n = get_height()
    for i in range(n):
        for j in range(n):
            if j < n - i -1 :
                print(" ", end="")
            else:
                print("#", end="")
        print()
  
         
    

def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9 :
            break
    return n
            
main()
            

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int get_positive_int (void);
int main (void)
{
  int i = get_positive_int ();

  for (int l = 1 ; l < i+1 ; l++)
  {
     for (int j = 0 ; j < i ; j++)
     {
        if (j < i-l)
        {
            printf (" ");
        }
        else
        {
            printf ("#");
        }
     }
     printf ("\n");
  }
}




int get_positive_int (void)
{
    int n;
    do
    {
        n = get_int ("Height:");
    }
    while (n<1 || n>8);
    return n;
}